inst/WORDLIST.txt

```diff
@@ -1,1246 +1,1255 @@
+tribble
+trt
+trtemfl
+trunc
+ttae
+tte
+ubuntu
+ucla
+ug
+ui
+un
+ungroup
+unlist
+unv
+upskilled
+upskilling
+useNA
+useR
+ut
+utm
+valuelevel
+vehIoJgdA
+venkatachalam
+vslb
+waddella
+wasm
+wayback
+webR
+webassembly
+webm
+webp
+webr
+wfk
+wg
+wikimedia
+wikipedia
+writeLines
+writeup
+www
+xWEGJx
+xejEvfwk
+xlab
+xlsx
+xportr
+xpt
+yaml
+yay
+ylab
+ylim
+ymd
+yml
+youtu
+youtube
+ytJ
+yv
+yyyy
+zMachine
+zelos
+zhu
+zt
+ztamazonaws
+zxqguo
+Żyła
+adrg
+amazonaws
+encyclopaedic
+
+zt
+Khatri
+Leena
+tm
+tmc
+css
+AJu
+xWEGJx
+andre
+drodziewicz
+eWSIK
+ORobErYX
+pharmar
+riskmetric
+Rodziewicz
+sa
+Sanofi's
+shinyconf
+ShinyConf
+xejEvfwk
+eco
+cfc
+deaad
+img
+src
+svg
+wikimedia
+de
+RightTool
+Rlogo
+sessioninfo
+valuelevel
+writeup
+asis
+Pawel
+py
+ropensci
+Rucki
+runiverse
+thecoatlessprofessor
+strsplit
+yaml
+YAML
+PARCAT
+SRCDOM
+SRCSEQ
+SRCVAR
+trt
+TRTEDTM
+trtemfl
+TTAE
+ttae
+tte
+ug
+admiralmetabolic
+admiralpeds
+AEN
+BMS
+cdc
+growthcharts
+frontrunner
+hc
+lgth
+TBC
+ek
+tinyurl
+anthropometric
+SDS
+CRO
+deps
+learnings
+pkgdown
+upskilled
+admiralci
+Patil
+pkgs
+Cytel
+ADRS
+ADOE
+ADBCVA
+ADVFQ
+IMWG
+strived
+admiralonco
+admiralophtha
+difftime
+POSIXct
+POSIXt
+adrian
+Alonso
+Amor
+Coursera
+Genentech
+Genentech's
+jlaxamana
+khatri
+Laxamana
+leena
+ne
+Stoilova
+tamor
+Tatiana
+TLF
+trainings
+upskilling
+coursera
+ADT
+a
+Céline
+cpp
+DALL
+DfReader
+DSJSON
+jpeg
+Piraux
+targetdatatype
+TDF
+TestDataFactory
+ACTARM
+AETERM
+barchart
+Berel
+dataname
+datanames
+Demin
+Dror
+initializer
+nolint
+pak
+Pavel
+shinyApp
+sparkline
+strived
+accountabilities
+curation
+requestor
+pageId
+viewpage
+Astrazeneca
+Astrazeneca’s
+autistics
+biometrics
+Cytel's
+groupthink
+Needleman
+neurodivergent
+Neurodiversity
+NK
+PharmaSUG
+astrazeneca
+laura
+MeetLaura
+needleman
+Alexandros
+clusterSetRNGStream
+CMRG
+doFuture
+elbersb
+filelock
+HPC
+Kouretsis
+L'Ecuyer
+mirai
+nanonext
+opre
+parallelization
+Parallelization
+parallelizes
+parallelMap
+pubsonline
+reprovision
+shikokuchuo
+tensio
+tidylog
+tidyr
+wlandau
+zzz
+axecute
+readRDS
+saveRDS
+WEL
+Parmar
+RMarkdown
+roxy
+roxygen
+Voilà
+examplesShinyLive
+iframe
+lzstring
+mypackage
+parmsam
+shinyl
+zzz
+EG
+interpretability
+AMBUL
+chr
+Dura
+ECGINT
+eg
+EGBLFL
+egdtc
+EGDTC
+EGDY
+EGELTM
+EGLOC
+EGORRES
+EGORRESU
+EGSEQ
+EGSTAT
+EGSTRESC
+EGSTRESN
+EGSTRESU
+EGTEST
+EGTESTCD
+EGTPT
+EGTPTNUM
+EGTPTREF
+reb
+Shuliar
+stringsAsFactors
+VISITNUM
+Vladyslav
+dataset's
+BEFORE’
+CDASH
+CMSTRTPT
+CMTRT
+CRF
+DM
+eCRF
+EDC
+eDT
+FLUIDS’
+hardcode
+hardcoded
+MDPRIOR
+MHOCCUR
+MHPRESP
+MHTERM
+NonCRF
+oak
+Pressure
+RELREC
+Roadmap
+SV
+TEMP
+VSMETHOD
+VSPOS
+VSTEST
+VSLAT
+tgt
+Y
+oakgarden
+stringsAsFactors
+Rammprasad
+mmHg
+MDRAW
+kable
+Ganapathy
+FASCAT
+DOSU
+dat
+CORTISPORIN
+CMSTTPT
+CMMODIFY
+CMDOSU
+CMDOSE
+clst
+br
+Varialble
+sdtmoak
+Ercan
+Suekuer
+aNCA
+NCA
+nca
+admiralverse
+dverse
+datatable
+ezVpsadSF
+lepore
+Lepore
+mauro
+maurolepore
+VhmnADBDXv
```

Comment:
Strangely, running CICD.R locally produced different results than what I see on GitHub. To work around that I edited inst/WORDSLIST.txt manually. 

Have you experience something similar before? Any tip you can share?

The weird words "ezVpsadSF", and "VhmnADBDXv" belong to the URL of YouTube videos.

Reply:
Yeah as the blog post repo is not structured as a package (yet), CICD is a bit improvised. 
Local results probably depend on your package version, and we had trouble before with that as well. 
Does it work for you to check the CICD results and implement changes based on that?

Reply:
PS: I really liked the video you did, did not read the post yet :)

Reply:
Thanks @StefanThoma! 

I'm glad you liked the video :-)

--

> Does it work for you to check the CICD results and implement changes based on that?

Yes, that worked for me. My local run detected fewer words than the run on GH  Actions, triggering an informative failure that I then used to update the words list manually. The result passed.

For example, see this [failing run](https://github.com/pharmaverse/blog/actions/runs/12223540652/job/34095197187) showed exactly what I needed to add to inst/WORDSLIST.txt.

```
Run testthat::test_that(desc = "Check spelling",
  WORD          FOUND IN
datatable     document_your_universe_with_dverse.qmd:121,[13](https://github.com/pharmaverse/blog/actions/runs/12223540652/job/34095197187#step:5:14)3,148
ezVpsadSF     document_your_universe_with_dverse.qmd:159
lepore        document_your_universe_with_dverse.qmd:165
Lepore        document_your_universe_with_dverse.qmd:4
mauro         document_your_universe_with_dverse.qmd:165
maurolepore   document_your_universe_with_dverse.qmd:[15](https://github.com/pharmaverse/blog/actions/runs/12223540652/job/34095197187#step:5:16)4,163
VhmnADBDXv    document_your_universe_with_dverse.qmd:[16](https://github.com/pharmaverse/blog/actions/runs/12223540652/job/34095197187#step:5:17)0
── Failure: Check spelling ─────────────────────────────────────────────────────
...
```